src/modules.json

```diff
@@ -293,6 +294,9 @@
       "js_file": "js/net.js",
       "require": ["assert", "events", "stream", "tcp", "util"]
     },
+    "internal_process_next_tick": {
```

Comment:
Why not `internal/process/next_tick'?

Reply:
right now module names are stored in cmake variable, which could not include slash "/" nor back slash"\".